source/src_lcao/gint_k_rho.cpp

```diff
@@ -8,6 +8,14 @@
 #include "../module_base/blas_connector.h"
 #include "global_fp.h" // mohan add 2021-01-30
 
+#ifdef _OPENMP
+#include <omp.h>
+#endif
+
+#ifdef __MKL
+#include <mkl_service.h>
```

Comment:
Better use `mkl.h` here. For the official usage of `mkl_set_num_threads` and functions like that, only `mkl.h` is included.
Besides, using `""` instead of `<>` for non-system headers.

Reply:
only mkl_service has some function such as mkl_get_max_threads, it seems useful for openmp